README.md

```diff
@@ -445,7 +445,7 @@ $ git clone https://github.com/BurntSushi/ripgrep
 $ cd ripgrep
 $ cargo build --release
 $ ./target/release/rg --version
-0.1.3
+14.1.0
```

Comment:
Please remove this change. It's just an example. It is an intentionally old version. I want fewer places where I need to update the version, not more.

Reply:
Okay sure thing, I also submitted another pull request which changes the link of the releases section to the latest one, I apologize if that wasn't meant to be updated either.

Reply:
What? You only have one PR open: https://github.com/BurntSushi/ripgrep/pulls/PrathamVerma999

No need to apologize. :)

Reply:
oh it's this one
https://github.com/BurntSushi/ripgrep/commit/5645f7a2981421ce3f0905a8b4df15187caa9670
I accidentally pushed them in the same pr, I'm new to contributing sorry. is there a way for you to just merge one commit if you wanted to or do I need to make another pull request. Even if you don't merge it please let me know so I don't make this mistake in the future.

Reply:
Ideally you would just rebase the branch and drop the commits you don't want to include. Following some "git rebase tutorial" might help.

Reply:
sure thank you so much I'll do that by tonight